README.md

```diff
@@ -38,3 +38,33 @@ pytest tests/test_scanner.py
 3. Click “Run 'pytest in tests'”
 
 4. Or click the green ▶ next to a test function
+
+## Interpreter
+
+### Principles for List Handling
```

Comment:
Please review the entire README for what should and should not be here (given its relevancy to this PR).